config/crds/clusterlink.net_imports.yaml

```diff
@@ -36,8 +36,9 @@ spec:
             description: Spec represents the attributes of the imported service.
             properties:
               lbScheme:
-                description: LBScheme is the load-balancing scheme to use (e.g., Random,
-                  Static, ECMP)
+                default: ecmp
+                description: LBScheme is the load-balancing scheme to use (e.g., random,
```

Comment:
q: if the key is called `lbScheme` in line 38, why refer to `LBScheme` in the description?

Reply:
This is an auto-generated file